core/src/html/layout.rs

```diff
@@ -666,48 +666,51 @@ impl<'a, 'gc> LayoutContext<'a, 'gc> {
             // return new_empty_font(context, span, self.font_type);
         }
 
-        // Check if the font name is one of the known default fonts.
-        if let Some(default_font) = match font_name.deref() {
-            "_serif" => Some(DefaultFont::Serif),
-            "_sans" => Some(DefaultFont::Sans),
-            "_typewriter" => Some(DefaultFont::Typewriter),
-            "_ゴシック" => Some(DefaultFont::JapaneseGothic),
-            "_等幅" => Some(DefaultFont::JapaneseGothicMono),
-            "_明朝" => Some(DefaultFont::JapaneseMincho),
-            _ => None,
-        } {
-            if let Some(&font) = context
-                .library
-                .default_font(
-                    default_font,
-                    span.style.bold,
-                    span.style.italic,
-                    context.ui,
-                    context.renderer,
-                    context.gc_context,
-                )
-                .first()
-            {
+        // Specifying multiple font names is supported only for device fonts.
+        let font_names: Vec<&str> = font_name.split(",").collect();
```

Comment:
Would this work if you iterated on the split without collecting into a Vec?

Reply:
I could do that, sure, but then I'm using the vec to get the fallback font name and IMO it's clearer when I do `font_names.first()` compared to not collecting into a vec and doing something else.